config/environments/production.rb

```diff
@@ -119,5 +119,15 @@
                  port: Settings.redis.port, drive: :hiredis }
   }
 
+  config.middleware.insert_before 0, 'Rack::Cors', logger: (-> { Rails.logger }) do
+    allow do
+      origins /[a-z0-9]+\.sumofus\.org/Z/i
```

Comment:
Yes and yes. This PR has been put on the back-burner, and this line of code in particular is an untested sketch. I'm actually curious what ideas people have for ways to test this block, given it's in production.rb and tests requests from other domains.